src/components/home/HomePageCard.module.css

```diff
@@ -0,0 +1,25 @@
+.card {
```

Comment:
Doporučuju do webpack přidat SASS loader, abyste mohli misto `.css` používat `.scss`, to vám hodně zjednodušší práci se stylováním.